### PR TITLE
Add num attribute on Lesson class

### PR DESCRIPTION
**Pull request by tarneaux**
Sometimes it happens you struggle to know which one of the modifications is the last one.
The pronotepy.dataClasses.Lesson.num can be used to know it: Like this you can just show the last modification.
The biggest num is for the latest modification. 
It comes from the `P` attribute:
[screenshot](https://ibb.co/SNCkHgH)

### CODE REVIEW

File: pronotepy/dataClasses.py [hunk 1]
```diff
@@ -376,7 +379,8 @@ class Lesson:
         'estRetenue': ('detention', bool, False),
         'estSortiePedagogique': ('outing', bool, False),
         'dispenseEleve': ('exempted', bool, False),
-        'listeVisios,V': ('virtual_classrooms', lambda l: [i["url"] for i in l], [])
+        'listeVisios,V': ('virtual_classrooms', lambda l: [i["url"] for i in l], []),
+        'P': ('num', str)
```
Comment (bain3) [inline pronotepy/dataClasses.py:383]: Possibly change to int? Does it have to be a string?

**Review by bain3 — COMMENTED** [review]
Maybe change the num type to int? Doesn't look like it has to be a string.

**Review by bain3 — APPROVED** [review]
LGTM!

File: pronotepy/dataClasses.py (hunk 1 above)
Comment (tarneaux) [reply pronotepy/dataClasses.py]: Yes, messed up here.
I changed it.

**Pull request merged**